library/spiders/books.py

```diff
@@ -0,0 +1,87 @@
+import scrapy
+from scrapy import Selector
+from scrapy.http import Response
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+from selenium.webdriver.common.by import By
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def __init__(self):
+        super().__init__()
+        self._driver = self.__set_chrome_driver()
+        self._rating_table = {
+            "Zero": 0,
+            "One": 1,
+            "Two": 2,
+            "Three": 3,
+            "Four": 4,
+            "Five": 5
+        }
+
+    def __set_chrome_driver(self):
+        options = Options()
+        options.add_argument("--headless=new")
+        return webdriver.Chrome(options=options)
+
+    def close(self, reason: str):
+        self._driver.quit()
```

Comment:
The `close` method correctly quits the Selenium driver, which is important to release resources. Ensure this method is called appropriately when the spider finishes.